### ISSUE TITLE
Proxyman stops showing simulator requests coming from a specific URL after a while

### ISSUE THREAD

**Issue by VladIacobIonut** [ProxymanApp/Proxyman#1696]
## Description
After downloading Proxyman, installing required certificates and starting to intercept traffic from the iOS simulator everything goes well seemingly for the first session of the app. I can see requests from my app with the base URL for our server, decode them and map local. 

It happened to me for the second time now that after some time (~ 1 day), I cannot see traffic coming from my app's base url. Proxyman is intercepting request from the analytics framework and other apps in the simulator (like the News app). 

The first time this was solved by deleting Proxyman and installing it again. Its the second time this happens now and I guess there's probably got to be a solution quicker than the first one.

Proxyman stops showing requests coming from a specific URL.

## Expected Behavior

Proxyman shows requests coming from a specific URL normally.

## Environment

- App version:  4.8.1
- macOS version: Ventura

**Comment by VladIacobIonut** [comment]
Does it have to do with **Free version** plan limitations maybe, as in limiting the requests to a certain limit?

**Comment by NghiaTranUIT** [comment]
@VladIacobIonut it's a bug from iOS Simulator. Some users have experienced this bug on iOS 16 Simulator too. Ref: https://github.com/ProxymanApp/Proxyman/issues/1268#issuecomment-1238595340

They confirmed that this issue happened with Charles proxy too. Using Atlantis or iOS Physical device will fix it.

**Comment by VladIacobIonut** [comment]
@NghiaTranUIT yup can confirm. everything works correctly with iOS 14.5 sim. thank you!
